pygmt/src/surface.py

```diff
@@ -68,7 +73,50 @@ def surface(data=None, x=None, y=None, z=None, **kwargs):
     outgrid : str
         Optional. The file name for the output netcdf file with extension .nc
         to store the grid in.
-
+    convergence : float
```

Comment:
Can also be string if appending the `%` sign to the number.
```suggestion
    convergence : float or str
```